source/Ship.cpp

```diff
@@ -193,7 +193,9 @@ void Ship::Load(const DataNode &node)
 					? "no key." : "key: " + child.Token(1)));
 			continue;
 		}
-		if(key == "sprite")
+		if(key == "shipyard display name" && child.Size() > 1)
+			shipyardDisplayName = child.Token(1);
```

Comment:
Why only in the shipyard?

Reply:
~Because I don't want to go through everywhere else and change stuff right now.~
If a person wants to sell multiple different configurations of a ship where the only changes are outfits, it doesn't make sense for the model name to be replaced with something else elsewhere, since the initial outfit configuration is the only difference.

Reply:
Then perhaps both a custom display name and a custom shipyard name are necessary.

Reply:
Should that replace `modelName` everywhere it is present?